src/Symfony/Component/Mailer/SMimeRecipient.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer;
+
+use Symfony\Component\Mime\Address;
+
+/**
+ * @author Fabien Potencier <fabien@symfony.com>
+ */
+final class SMimeRecipient extends Address
```

Comment:
this requires bumping the min version of the `mime` component to be the one providing a non-final class (which is a change warranting a changelog entry in the mime component btw).